release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2550,6 +2602,78 @@ and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bu
 
 // TODO: ^^^ Add BZ when created.
 
+[id="ocp-4-10-ran-ga-known-issues"]
+
+* The Kubelet service monitor scrape interval is currently set to a hard-coded value. This means that there is less available CPU resources for workloads.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035046[*BZ#2035046*])
+
+* Deploying a single node Openshift cluster for vRAN Distributed Units can take up to 4 hours.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2035036[*BZ#2035036*])
+
+* Currently, if an {rh-rhacm} policy was enforced to a target cluster, you can create a `ClusterGroupUpgrade` CR including that enforced policy again as a `managedPolicy` to the same target cluster. This should not be possible.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2044304[*BZ#2044304*])
+
+* If a `ClusterGroupUpgrade` CR has a `blockingCR` specified, and that `blockingCR` fails silently, for example if there is a typo in the list of clusters, the `ClusterGroupUpgrade` CR is applied even though the `blockingCR` is not applied in the cluster.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042601[*BZ#2042601*])
+
+* If a `ClusterGroupUpgrade` CR validation fails for any reason, for example, because of an invalid spoke name, no status is available for the `ClusterGroupUpgrade` CR, as if the CR is disabled.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2040828[*BZ#2040828*])
+
+* Currently, for `ClusterGroupUpgrade` CR state changes, there is only a single condition type available - `Ready`. The `Ready` condition can have a status of `True` or `False` only. This does not reflect the range of states the `ClusterGroupUpgrade` CR can be in.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2042596[*BZ#2042596*])
+
+* The alpha channel for the PTP Operator reports an error that prevents a `PtpConfig` CR from being applied when the `summary_interval` value does not match the value set for `logSyncInterval`. To work around this issue, update all reference PTP configurations to set `summary_interval` and `logSyncInterval` to -4.
```

Comment:
The RC for this issue is already verfieid in ptp 4.10 via bz 2055371. The workaround is no longer needed unless customer is not using the 4.10 ptp operator